db/controller/commentsIdDelete.controller.js

```diff
@@ -0,0 +1,20 @@
+const { deleteCommentById } = require("../models/commentsIdDelete.model");
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+
+  deleteCommentById(comment_id)
+    .then((deletedRowCount) => {
+      if (deletedRowCount === 0) {
+        return res.status(404).json({ error: "Comment not found" });
+      }
+      res.status(204).send();
+    })
+    .catch((err) => {
+      if (err.code === "22P02") {
+        res.status(400).json({ error: "Invalid comment ID format" });
```

Comment:
This error should be passed using next to you error handling middle-wear to be handled by the psql error handler